components/branding/InfoBlock.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { Typography } from '@mui/material'
+import { useTheme } from '@mui/material/styles'
+import FileCopyIcon from '@mui/icons-material/FileCopy';
```

Comment:
i don't love the file imagery since we're not copying a file. consider alternative icons like MUI's ContentCopy, CopyAll, or ContentPaste.